src/erlcloud_sns.erl

```diff
@@ -171,17 +173,17 @@ confirm_subscription(SnsEvent, Config) ->
     TopicArn = binary_to_list(proplists:get_value(<<"TopicArn">>, SnsEvent, <<>>)),
     confirm_subscription2(Token, TopicArn, Config).
 
--spec confirm_subscription/3 :: (sns_event(), string(), string()) -> string().
+-spec confirm_subscription/3 :: (sns_event(), parameter(), parameter()) -> string().
 confirm_subscription(SnsEvent, AccessKeyID, SecretAccessKey) ->
     confirm_subscription(SnsEvent, new_config(AccessKeyID, SecretAccessKey)).
 
 
 
--spec confirm_subscription2/2 :: (string(), string()) -> string().
+-spec confirm_subscription2/2 :: (parameter(), parameter()) -> string().
 confirm_subscription2(Token, TopicArn) ->
     confirm_subscription2(Token, TopicArn, default_config()).
 
--spec confirm_subscription2/3 :: (string(), string(), aws_config()) -> string().
+-spec confirm_subscription2/3 :: (parameter(), parameter(), aws_config()) -> string().
```

Comment:
Line 186 is too long: -spec confirm_subscription2/3 :: (parameter(), parameter(), aws_config()) -> string()..